### PR TITLE
Fix failing tests

### LINKED ISSUE #1032: cypress: Find a better way to assert node content
https://github.com/aeternity/superhero-wallet/blob/d02410462c1d3d9e31d50f92f1214791c97803b1/tests/e2e/integration/amount-input.js#L8

https://github.com/aeternity/superhero-wallet/blob/d02410462c1d3d9e31d50f92f1214791c97803b1/tests/e2e/integration/amount-input.js#L12

Consider using should instead as far as it is "automatically retried until they pass or time out" (https://docs.cypress.io/api/commands/should).

### PR DESCRIPTION
**Pull request by CedrikNikita**
fixed #1032

### CODE REVIEW

**Review by etharner — APPROVED** [review]

File: tests/e2e/integration/transactions.js [hunk 1]
```diff
@@ -1,42 +1,20 @@
...
-  });
-
   it('Load transactions, load additional transactions on scroll', () => {
-    cy.openTransactions().get('[data-cy=list]').children().should('have.length', TXS_PER_PAGE);
+    cy.login()
+      .openTransactions()
+      .get('[data-cy=list]')
+      .children()
+      .its('length')
+      .should('be.gte', 0);
```
Comment (davidyuk) [inline tests/e2e/integration/transactions.js:10]: isn't the length of anything is always greater than or equal to 0?
```suggestion
      .should('be.gt', 0);
```

File: tests/e2e/integration/amount-send.js [hunk 2]
```diff
@@ -1,7 +1,9 @@
 describe('Test cases AmountSend component', () => {
   it('Calculates currency on enter amount, validates entered amount, shows correct balance', () => {
+    // eslint-disable-next-line cypress/no-unnecessary-waiting
     cy.login()
       .openWithdraw()
+      .wait(1000)
       .get('[data-cy=balance-currency]')
       .invoke('text')
       .then((text) => expect(text).not.to.eq('$0.00'))
```
Comment (davidyuk) [inline tests/e2e/integration/amount-send.js:9]:
```suggestion
    cy.login()
      .openWithdraw()
      .get('[data-cy=balance-currency]')
      .invoke('text')
      .should('not.equal', '$0.00')
```

File: tests/e2e/integration/networks.js [hunk 3]
```diff
@@ -84,6 +85,7 @@ describe('Test cases for networks page', () => {
       .should('contain', defaultNetwork.middlewareUrl)
       .goBack()
       .goBack()
+      .wait(2500)
       .get('.transaction-list')
       .should('exist')
```
Comment (davidyuk) [inline tests/e2e/integration/networks.js:90]: I can't get why `wait` is needed here.
> Create an assertion. Assertions are automatically retried until they pass or time out.
https://docs.cypress.io/api/commands/should#Syntax

File: tests/e2e/integration/transactions.js (hunk 1 above)
Comment (CedrikNikita) [reply tests/e2e/integration/transactions.js]: That is obviously a mistake, yes

File: tests/e2e/integration/networks.js (hunk 3 above)
Comment (CedrikNikita) [reply tests/e2e/integration/networks.js:88]: The goal of this PR was to find the weak spots, and prove them by restarting test with empty commits. That's why there were status/hodl label, this was not the final stage of development (intentionally). You are right it's is better to increase timeout for this get.

File: tests/e2e/integration/amount-send.js (hunk 2 above)
Comment (CedrikNikita) [reply tests/e2e/integration/amount-send.js]: It won't work since text are not trimmed.
https://github.com/cypress-io/cypress/issues/3887

**Pull request merged**